<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LauncherWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditQExePath">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Quake Executable Full Path</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;C:\Quake\quakespasm.exe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFindQExe">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to locate Quake Executable</string>
    </property>
    <property name="text">
     <string>Find Quake Executable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLaunchExe">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch Quake Executable</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Mod options</string>
    </property>
    <widget class="QRadioButton" name="radioButtonRogue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dissolution of Eternity</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonHipnotic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Scourge of Armagon</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCustomGame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom Mods</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditModDir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Mod folder name, e.g. &quot;Nehahra&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFindMod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click to select mod folder</string>
     </property>
     <property name="icon">
      <iconset theme="applications-system"/>
     </property>
    </widget>
    <widget class="QLabel" name="labelMap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Map</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditMap">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Start map name</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEditFullString">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxDisplay">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Display options</string>
    </property>
    <widget class="QLabel" name="labelResolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxResolution">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFullScreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Screen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelBPP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>bpp</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBPP">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVSync">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow VSync</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAspect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow Aspect Ratio Correction</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelFreq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>freq</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFreq">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonQSync">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Sync Quaddicted</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
